protos/xyz/block/ftl/v1/console/console.proto

```diff
@@ -151,6 +152,7 @@ message TimelineQuery {
   }
 
   repeated Filter filters = 1;
+  int32 limit = 2;
```

Comment:
I tried making this optional but for some reason, it was always nil even if I passed it in a `GetTimeline` request from the console. I couldn't figure out how to resolve that so I removed the optionality. Not sure what's up there.